src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,10 @@
+# Logging settings for unit tests
+
+# The root logger with appender name
+rootLogger=DEBUG, STDOUT
+
+# Assign STDOUT a valid appender & define its layout
+appender.console.name=STDOUT
+appender.console.type=Console
+appender.console.layout.type=PatternLayout
+appender.console.layout.pattern=[%highlight{%p}{INFO=blue}] [%c{1}] %m%n
```

Comment:
If possible, I would like to match the bright white for method name if possible.  I started reading the highlight rules, but I got very confused. 🤕